wordpress-mobile/release-toolkit
# Integrate screenshot tool

**To Test:**

1) Generate a set of Android screenshots (after https://github.com/wordpress-mobile/WordPress-Android/pull/9782 is merged) using `bundle exec fastlane android screenshots` (or ping me if you'd like a premade copy of them to test this PR with).
2) Ensure that the `fastlane/metadata` directory contains the new screenshots.
3) From the `develop` branch, run `bundle exec fastlane android create_promo_screenshots`. It'll run this code, and will generate localized screenshots into the `fastlane/promo_screenshots` directory.

**Prerequisites:**
- Ensure you've installed the complete Noto Sans font family on your local machine: https://fonts.google.com/specimen/Noto+Sans (todo: this can be automated in the future)
